index.html

```diff
@@ -34,6 +34,12 @@
         >
```

Comment:
UX suggestion: filters checkbox and text don't look aligned. Maybe could be vertically aligned.
![image](https://user-images.githubusercontent.com/91944618/154856906-0c65f501-c327-4d0b-9c5f-67abef15fc7b.png)